packages/flutter/test/widgets/binding_test.dart

```diff
@@ -141,7 +155,9 @@ void main() {
     WidgetsBinding.instance.handlePopRoute();
     WidgetsBinding.instance.handlePushRoute('/');
     WidgetsBinding.instance.handleRequestAppExit();
-    expect(observer.removeSelf(), greaterThan(11));
+    await tester.idle();
+    expect(observer.removeSelf(), greaterThan(1));
```

Comment:
Did you mean to change this from 11 to 1?

Reply:
I don't want to use an exact value cause this goes up or down pretty quickly depending on how many methods get called. I think it's probably ok to just make sure it got added more than in the constructor, so I put it to 1.